src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -632,8 +630,7 @@ private static XDocument CreateMethodCallSnippet(string methodName, bool include
                 declarations.Add(new XElement(
                     snippetNamespace + "Literal",
                     new XElement(snippetNamespace + "ID", new XText(parameter.Name)),
-                    new XElement(snippetNamespace + "Function", new XText($"ArgumentValue({SymbolKey.CreateString(parameter, cancellationToken)})")),
-                    new XElement(snippetNamespace + "Default", new XText(""))));
+                    new XElement(snippetNamespace + "Default", new XText(parameterValues.GetValueOrDefault(parameter.Name, "")))));
```

Comment:
⚠️ Make sure to test the results with special characters. A string literal should suffice in an integration test provided it contains:

* The character `<`
* The character `&`
* Two consecutive spaces
* The text `$placeholder$`
* The text `$end$`

Reply:
@sharwell Any suggestions for actually writing that? I did test the consecutive spaces by hacking in a argument provider that would return `"    "` for a string argument, but that's not something I can do as easily in integration tests.

Reply:
One of the integration tests writes a string argument, and then changes overloads away and back to it. You can copy that test, reduce it, and then make a `[Theory]` so it can be executed with several different strings of interest.

Reply:
Ah, right, that'll work.

Reply:
Test added, except for the < case which is broken because of https://github.com/dotnet/roslyn/issues/29669.